includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -524,7 +524,17 @@ private function process_webhook_dispute_created( $event_body ) {
 		$dispute_id   = $this->read_webhook_property( $event_object, 'id' );
 		$charge_id    = $this->read_webhook_property( $event_object, 'charge' );
 		$reason       = $this->read_webhook_property( $event_object, 'reason' );
-		$order        = $this->wcpay_db->order_from_charge_id( $charge_id );
+		$amount_raw   = $this->read_webhook_property( $event_object, 'amount' );
+		$evidence     = $this->read_webhook_property( $event_object, 'evidence_details' );
+		$due_by       = $this->read_webhook_property( $evidence, 'due_by' );
+
+		$order = $this->wcpay_db->order_from_charge_id( $charge_id );
+
+		$currency = $order->get_currency();
+		$amount   = wc_price( WC_Payments_Utils::interpret_stripe_amount( $amount_raw, $currency ), [ 'currency' => strtoupper( $currency ) ] );
```

Comment:
This currency format comes out different to that used for the order success note. E.g. `110,00 € EUR` vs `110,00 €`

Order success note:
<img width="273" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/d8d1e892-a0e7-41c0-a301-24744a30a53b">

Dispute note:
<img width="272" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/4e039fd3-7850-41b5-828f-40b1cc58b753">


What do you think about using the same method ([`WC_Payments_Order_Service::get_order_amount()`](https://github.com/Automattic/woocommerce-payments/blob/8710fc79037796965e469b448c49cbc0a7782d4a/includes/class-wc-payments-order-service.php#L782)) to get the formatted currency string:

https://github.com/Automattic/woocommerce-payments/blob/8710fc79037796965e469b448c49cbc0a7782d4a/includes/class-wc-payments-order-service.php#L782

Reply:
Tricky … not sure how to call this one, so leaving as is for now. Tracing how that value is rendered, I see it uses `WC_Payments_Explicit_Price_Formatter::get_explicit_price()`. The method not quite right for my purposes – it depends on order and pulls currency from order (we need to use dispute, even though it should be the same), so would need to refactor to remove dependency on order.

Pros of current approach:

- Deferring to WooCommerce core for rendering prices. This should be more consistent overall. E.g. respect user preferences for number formatting. 
- Less code impact (though … maybe we should dig into `WC_Payments_Explicit_Price_Formatter` as a follow up - see below). 

Cons of current approach:

- Inconsistent price formats between WCPay order notes.

Which makes me wonder - is there a bug / arbitrary difference in `WC_Payments_Explicit_Price_Formatter::get_explicit_price()`? I see in your screenshot there's a different decimal point, plus I don't understand why it suffixes currency symbol.

Reply:
I'll retest with a multicurrency setup, want to ensure this is working correctly in that case. For example USD and CAD could be confusing.

Reply:
Interesting! Some additional examples:

**WCPay disputes screens** display EURO amounts as `€110.00 EUR` with a decimal not comma and show `EUR`: 

Disputes List
<img width="344" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/8e5c3d0d-ea08-42f6-bd21-99a0af1e549d">

Dispute details
<img width="398" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/73f6c6cf-756b-4791-9757-422af64d0e3e">

The **WC order screen total** shows as `110.00 € EUR` with a decimal.
<img width="284" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/ab56c77c-ab8a-499b-a9d0-1446faa45bea">

Reply:
We need the currency consistently displayed in both order notes, so I'm fixing it. Will probably refactor `WC_Payments_Explicit_Price_Formatter` so there's a simple render price function I can use.

<img width="407" alt="Screenshot 2023-06-07 at 2 34 59 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4167300/8a9b7aee-91f5-4ebf-b79a-7d7da0853c38">

Reply:
I've done a light refactor so we can use `WC_Payments_Explicit_Price_Formatter` to append currency symbol when needed. 3887d90461f744d6262df99c83b9c601200b2da6